crates/cli-tools/src/changelog.rs

```diff
@@ -253,12 +316,19 @@ pub async fn execute_ci() -> Result<()> {
 }
 
 /// Updates a CHANGELOG.md file and CHANGELOG.md files of dependencies.
-pub async fn execute_change(path: &str, _severity: &Severity, _description: &str) -> Result<()> {
-    ensure!(fs::exists(path).await, "Crate does not exist: {path}");
+pub async fn execute_change(path: &str, severity: &Severity, description: &str) -> Result<()> {
+    let changelog_file_path = format!("{path}/CHANGELOG.md");
+
+    ensure!(fs::exists(&changelog_file_path).await, "Crate does not exist: {changelog_file_path}");
```

Comment:
This is actually redundant. The `read_file` function below will fail if the file does not exist. And the `changelog_file_path` can actually be inlined and removed.

Reply:
Remove the `ensure!` check, however keeping the changelog_file_path variable since it is used twice.

Something I've been thinking about is storing crate path as internal state to Changelog struct. Then don't need to pass it around.

Reply:
Yes, it's probably a good idea. And in that case we can inline `parse` in `read_file`, and rename `read_file` and `write_file` to `read` and `write`.

Reply:
Sounds good, I'll do that in a follow up PR right after this one. I prefer to keep the PR's at atomic as possible to make review easier. Not mixing old with new.